### ISSUE TITLE
Poll: Several galleries in one Colorbox

### ISSUE THREAD

**Issue by mahotilo** [Typesetter/Typesetter#447]
Since old times, all galleries on one page are combined in Colorbox. Therefore, I can start viewing images in one gallery and slide to images from other. Is it right?

**Comment by juek** [comment]
Right … wrong … who knows?
IMO it has more pros than cons the way it is. E.g. can we have split galleries and thus have several hundred images per page - sth. that doesn't work in a single one.

**Comment by mahotilo** [comment]
> IMO the way it is has more pros than cons.

Yes, it depends on the use. I prefer when the user can cycling through the gallery's images over and over again without going out to another gallery. E.g., if I started looking for "Ocean" wallpapers gallery, I would not want to go to the "City" gallery.

I have changed this behaviour, but it needs editing both Editing.PHP (added rel=... to <a>) and main.JS (deleted rel=... from colorbox options). I think, it is not easy to implement via plugin.

What will others say? Should we limit the colorbox group to the current gallery?
